src/ImGui.NET.SampleProgram.XNA/ImGuiRenderer.cs

```diff
@@ -215,29 +192,162 @@ protected virtual void UpdateInput()
+                // Skipping Keys.Sleep ~ no idea what it is.
+                // Numpad0 - Numpad9 already done above.
+                case Keys.Multiply: imguikey = ImGuiKey.KeypadMultiply; return true;
+                case Keys.Add: imguikey = ImGuiKey.KeypadAdd; return true;
+                // Skipping Keys.Separator ~ no idea what it is.
+                case Keys.Subtract: imguikey = ImGuiKey.KeypadSubtract; return true;
+                case Keys.Decimal: imguikey = ImGuiKey.KeypadDecimal; return true;
+                case Keys.Divide: imguikey = ImGuiKey.KeypadDivide; return true;
+                // F1 - F12 already done above.
+                // ImGui don't have keys for F13 - F24.
+                case Keys.NumLock: imguikey= ImGuiKey.NumLock; return true;
+                case Keys.Scroll: imguikey = ImGuiKey.ScrollLock; return true;
+                case Keys.LeftShift:
+                case Keys.RightShift:
+                    imguikey = ImGuiKey.ModShift; return true;
+                case Keys.LeftControl:
+                case Keys.RightControl:
+                    imguikey = ImGuiKey.ModCtrl; return true;
+                case Keys.LeftAlt:
+                case Keys.RightAlt:
+                    imguikey = ImGuiKey.ModAlt; return true;
+                /*case Keys.BrowserBack
+                case Keys.BrowserForward:
+                case Keys.BrowserRefresh:
+                case Keys.BrowserStop:
+                case Keys.BrowserSearch:
+                case Keys.BrowserFavorites:
+                case Keys.BrowserHome:
+                case Keys.VolumeMute:
+                case Keys.VolumeDown:
+                case Keys.VolumeUp:
+                case Keys.MediaNextTrack:
+                case Keys.MediaPreviousTrack:
+                case Keys.MediaStop:
+                case Keys.MediaPlayPause:
+                case Keys.LaunchMail:
+                case Keys.SelectMedia:
+                case Keys.LaunchApplication1:
+                case Keys.LaunchApplication2:*/
+                case Keys.OemSemicolon: imguikey = ImGuiKey.Semicolon; return true;
+                case Keys.OemPlus: imguikey = ImGuiKey.Equal; return true;
+                case Keys.OemComma: imguikey = ImGuiKey.Comma; return true;
+                case Keys.OemMinus: imguikey = ImGuiKey.Minus; return true;
+                case Keys.OemPeriod: imguikey = ImGuiKey.Period; return true;
+                case Keys.OemQuestion: imguikey = ImGuiKey.Slash; return true;
+                case Keys.OemTilde: imguikey = ImGuiKey.GraveAccent; return true;
+                case Keys.OemOpenBrackets: imguikey = ImGuiKey.LeftBracket; return true;
+                case Keys.OemPipe: imguikey = ImGuiKey.Backslash; return true;
+                case Keys.OemCloseBrackets: imguikey = ImGuiKey.RightBracket; return true;
+                case Keys.OemQuotes: imguikey = ImGuiKey.Apostrophe; return true;
+                // case Keys.Oem8:
```

Comment:
@mellinoe feel free to add the keys which is missed, otherwise remove the commented line.